src/hooks/useFetch.js

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react"
+
```

Comment:
I'm not even sure what this code is doing! please explain in group.

Reply:
will do